### PR TITLE
fix(resources): images not showing

### LINKED ISSUE #254: [BUG] Multiple Broken images on the react framework site
**Describe the bug**
I have found several broken images on the react site. 

### Libraries
Screen Shot 2022-10-06 at 8 07 43 AM

### Tools
Screen Shot 2022-10-06 at 8 17 36 AM

### Courses
Screen Shot 2022-10-06 at 8 17 57 AM

### Podcasts
Screen Shot 2022-10-06 at 8 18 51 AM

### Books
Screen Shot 2022-10-06 at 8 19 15 AM

**Expected behavior**
Images should show up for each resource.

**Desktop (please complete the following information):**
 - OS: mac
 - Browser chrome

### PR DESCRIPTION
**Pull request by Olyno**
## Type of change

- [ ] Content addition
- [x] Bug fix
- [ ] Behavior change

## Summary of change

This pull request closes #254 by fixing image's urls.

## Checklist

- [x] This fix resolves #254 
- [x] The changes follow the [contributing guidelines](https://github.com/thisdot/framework.dev/blob/main/CONTRIBUTING.md)
- [x] I have verified the fix works and introduces no further errors

- [x] This change resolves #254 
- [x] I have confirmed with a maintainer that this change is desired and been
      given the go-ahead to work on it in the relevant issue discussion
- [x] The changes follow the [contributing guidelines](https://github.com/thisdot/framework.dev/blob/main/CONTRIBUTING.md)
- [x] I have verified the change and the rest of the site works as expected

### CODE REVIEW

File: packages/site/src/data/react/podcasts.ts [hunk 1]
```diff
@@ -63,7 +63,7 @@ export const podcasts: Podcast<typeof podcastTags[number]>[] = [
 	{
 		title: "React Round Up",
 		image:
-			"https://assets.fireside.fm/file/fireside-images/podcasts/images/4/45ac7cd2-9fff-46c9-842c-2d7e0d34cd4b/cover_small.jpg?v=1",
+			"https://topenddevs.nyc3.digitaloceanspaces.com/ktszo9706o6v5yqqy3s9sor1rgkq?response-content-disposition=inline%3B%20filename%3D%22reactroundup-1-scaled.jpeg%22%3B%20filename%2A%3DUTF-8%27%27reactroundup-1-scaled.jpeg&response-content-type=image%2Fjpeg&X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=STO4LZNRL5TU6SLCP4ZY%2F20221006%2Funused%2Fs3%2Faws4_request&X-Amz-Date=20221006T220027Z&X-Amz-Expires=300&X-Amz-SignedHeaders=host&X-Amz-Signature=bb8192309c86957c7578b49c3e1f8afebadb4a744a2ccdc111546a935d16385d",
```
Comment (jdwilkin4) [inline packages/site/src/data/react/podcasts.ts:66]: I am still seeing a missing image here
Screen Shot 2022-10-07 at 10 54 49 AM

When I try to go to the new link it says access denied
Screen Shot 2022-10-07 at 10 55 41 AM

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

File: packages/site/src/data/react/podcasts.ts (hunk 1 above)
Comment (Olyno) [reply packages/site/src/data/react/podcasts.ts:66]: Looks like they implemented this after my PR :/ Fixing it!

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**